### DISCUSSION TITLE
Green Software Verified Facts

### DISCUSSION THREAD

**Discussion by danielvaughan** [Green-Software-Foundation/discussions#47] [Community Working Group]
We have been preparing internal material on sustainability. We want to get three solid facts to present but it is hard to get facts that are verifiable.

For example suppose we want to make a statement about the carbon footprint of data centres.

There seems to be many figures and comparisons e.g. .They emit as much carbon as the aviation industry. However when you go looking for the source and the detail it seems it is easy to get a confused and inaccurate statement. For example:

- The subject of a statement can be the "Digital World", the ICT Industry or Date Centers
- The measure could be emissions or electricity consumption
- The timeframe can be now or a projection for the future

Therefore it is easy for someone in a talk to say: "Data centers produce 20% of the world's carbon emissions" where the original fact could have been something like "The Digital World is projected to use 20% of the world's electricity by 2025".

The risk is we alienate people with incorrect facts. I remember years ago we had an environment group at work that sent an email saying a TV left on standby uses as much electricity as when it is on. It ended up with a load of replies from people saying that was nonsense. In reality the original stat was probably something like a TV left on standby for 23 hours uses the same energy as being watched for an hour.

Could we maintain a repository of "GSF Facts" with citations that people could use safely in the knowledge that they can be backed up?

**Comment by adamj89** [comment]
Interesting @danielvaughan. I'm copying in @russelltrow here. This sounds like something that would compliment the Patterns project but also has elements of SOGS in it.

**Reply by adamj89** [reply]
Also added to CWG meeting for this week.

**Reply by NAMRATA-WOKE** [reply]
Thanks for adding it to the CWG call @adamj89.

**Comment by adamj89** [comment]
Parking for today and keeping this for the next call

**Reply by Willmish** [reply]
Maybe we could extend https://stateof.greensoftware.foundation/ to cover this?

**Comment by jawache** [comment]
SOGS was the attempt to solve this initially. We had several researchers examine data and surface the numbers that made the most sense. But I don't think it quite landed with the definitive list of facts.

I'd be comfortable with the GSF holding a repository of **recommended** facts as long as there is a clear **consensus** based approach for how those facts were added and a process that can be challenged. The word verified makes me nervous, we're not really verifying the facts. 

Most often all the facts are correct as you say, they just have different context but it's confusing to have so many. We can have our opinions on which ones we prefer and which ones we recommend. 

That language feels better to me as we are not really doing any investigative fact checking or peer review, redoing experiments etc...

**Reply by mistrc** [reply]
What are the next steps here?
